charts/MapTab.tsx

```diff
@@ -259,30 +344,17 @@ class MapWithLegend extends React.Component<MapWithLegendProps> {
                             }}
                         >
                             <span>
-                                {tooltipDatum
-                                    ? this.context.chart.map.data.formatTooltipValue(
-                                          tooltipDatum.value
-                                      )
-                                    : `No data for ${formatYear(
-                                          inputYear as number
-                                      )}`}
+                                {this.generateTooltip(tooltipDatum, inputYear)}
```

Comment:
I'm thinking if we should make the tooltip a little (functional) React component itself?

Reply:
resolved!